app/views/shared/_calendar_filter.html.erb

```diff
@@ -0,0 +1,12 @@
+<div id="resource-calendar-filter" class="resource-calendar-filter hide">
+  <%= label_tag(:filter_users, 'Filter by guider/group') %>
+  <%=
+    select_tag(
+      :filter_guiders,
+      grouped_options_for_select(guider_options),
+      multiple: 'multiple',
+      class: 't-users',
+      data: { module: 'advanced-select', placeholder: 'Guider or Group' }
```

Comment:
Had to add a placeholder as there is a bug in select2 which causes a JS error when using allowClear: true and having no placeholder

https://github.com/select2/select2/issues/3772